docs/api/routing.rst

```diff
@@ -394,6 +394,61 @@ A custom router is any class that implements the following interface:
 
             """
 
+Suffixed Responders
+-------------------
+
+While Falcon encourages the REST architectural style, it is flexible enough to accomodate other 
+paradigms. Consider the task of building an API for a calculator which can both add and subtract 
+two numbers. You could implement the 
+following:
+
+.. code:: python
+
+    class Add():
+        def on_get(self, req, resp):
+            resp.text = str(req.get_param_as_int('x) + req.get_param_as_int('y'))
+            resp.status = falcon.HTTP_200
+
+    class Subtract():
+        def on_get(self, req, resp):
+            resp.text = str(req.get_param_as_int('x) - req.get_param_as_int('y'))
+            resp.status = falcon.HTTP_200
+
+    add = Add()
+    subtract = Subtract()
+    api = falcon.API()
```

Comment:
Ah, sorry for a yet another late finding... As a matter of fact, [`falcon.API`](https://falcon.readthedocs.io/en/3.0.0b2/api/app.html#wsgi-app) is also deprecated :grimacing:

Reply:
Oh no, ha, I actually didn't know that. Will update

Reply:
Updated